### DISCUSSION TITLE
No automatic linebreak on comming soon page

### DISCUSSION THREAD

**Discussion by TS-R** [templaza/astroid-framework#184] [Q&A] [answered]
Hi, I am new to astroid framework and I have activated coming soon page until my page is finished. But on smaller screens like mobile phones I'm facing problems displaying my content text. The text is formated with `<p style="text-align: center;"><span style="background-color: #ffffff;">my text</span></p>`
By default automatic linebreak seams to be disabled by default. On small screen devices there whole text is in one line even if it doesn't fit with the screen width. If the text is to long it will displeyed compressed so the single words are overlapping each other. Can anybody help me to enable automatic linebreak for coming soon page content text?

Maybe this is a bug and I should report this as issue, too? Or is this expected behaviour?

Thank you in advance.

**Comment by Chacapamac** [comment, marked as answer]
I @TS-R , I will try to help.
Is your website reachable, it will be easier?
Did you try to clear all cache - Astroid/Joomla/Browser to be sure to see the right stuff. In my case, I did have to do it to see my formatting.

**I just test with Joomla 4, last Astroid and this is what I get.**
Seem all right for me...
![ok](https://user-images.githubusercontent.com/1907276/188270296-b3e136d0-4224-4d23-8ff0-563280fb98dc.jpg)
_**Note:** This is Firefox Developer IPhone 11 screen_

For sure, the “Coming Soon” text box in Astroid is plain text and you need to format it to your liking. On my test here, I add an H2, two double <br /> and a <strong> style.

**This is the structure of the page html you should get.**
![structure](https://user-images.githubusercontent.com/1907276/188270416-2030a336-5c1e-4166-902b-af8e4a0f2876.jpg)

**Try to add some css to the first container/up that get stretch by the text.**
I will try:
display:block
white-space: normal;
line-break: auto;

**Reply by TS-R** [reply]
Thank you very much for your fast replay.
Reseting the Astroid Cache was a good suggestion. Idisn't have it in mind even if I read it in the past. But this alsodidn't help.
Also the suggested CSS didn't change the behaviour.
For testing I have created a new child template and with this new template it worked as expected. After applying all main settings it still works. I don't know if I have damaged something or if something during the last Astroid update few days ago went wrong. 🤷🏼‍♂️
If I will have a little bit more time I will try to dive into it a I will see if Ican find the root-case and share it here.

Thank you very much for your support. 👍🏼

**Reply by Chacapamac** [reply]
• If you have a live website, I can take a look, if you want.

— Do you have the same HTML Structure as I shown?

Good Day!

**Comment by Chacapamac** [comment]
Happy to help!
Good Day!
